lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/util/Functions.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.util;
+
+import com.google.errorprone.annotations.FormatMethod;
+import io.trino.spi.TrinoException;
+
+import static io.trino.spi.StandardErrorCode.INVALID_FUNCTION_ARGUMENT;
+import static java.lang.String.format;
+
+public final class Functions
+{
+    private Functions() {}
+
+    @FormatMethod
+    public static void checkFunctionArgument(boolean condition, String message, Object... args)
```

Comment:
I don't think we would add an new shared utility for such a simple piece of code.  Instead, just inline this.

Reply:
I was simply following this https://github.com/trinodb/trino/blob/fb8cf33c76f3244fa99dd17dbcb6067e8cb719a8/lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/util/Procedures.java#L27
I find it quite feasible